src/components/GrowthChart/GrowthChartSelector/ChartSelectorDropdown/ChartSelectorDropdown.tsx

```diff
@@ -17,48 +17,36 @@ export const ChartSelectorDropdown = ({
     handleItemChange,
     isDisabled,
     dataTest,
-}: ChartSelectorDropdownProps) => (
-    <div className='flex flex-col'>
-        {isDisabled || items.length <= 1 ? (
-            <button
-                className='flex flex-row rounded border border-gray-300 py-1 gap-2 h-7 px-4 items-center whitespace-nowrap'
-                disabled
-                data-test={`${dataTest}-disabled-button`}
-            >
-                {title}
-            </button>
+}: ChartSelectorDropdownProps) => {
+    const tooltipContent = i18n.t('Gender is pre-selected based on the profile');
+    return (
+        isDisabled ? (
+            <Tooltip openDelay={500} closeDelay={50} content={tooltipContent}>
```

Comment:
Consider removing the const, as its only used in one place you might as well set the text string there. This also allows you to remove the return().

```suggestion
     (
        isDisabled ? (
            <Tooltip 
                openDelay={500} 
                closeDelay={50} 
                content={i18n.t('Gender is pre-selected based on the profile')}>
```